src/TestHeaders.am

```diff
@@ -5,21 +5,10 @@
 ## Please see the COPYING and CONTRIBUTORS files for details.
 ##
 
-## Test ./*.h files in a directory using test-suite/testheaders.sh
-
-## TODO: include in src/Common.am when all testHeaders users are the same.
-
 TESTS += testHeaders
 
-## .h dependency test script
-## aborts build process on errors; XXX: even with "make -k"
-testHeaders: $(srcdir)/*.h
-	$(SHELL) $(top_srcdir)/test-suite/testheaders.sh "$(CXXCOMPILE)" $^ || exit 1
+testHeaders: $(SOURCES) $(top_srcdir)/test-suite/testHeader.cc.in
```

Comment:
Can we move this dependency on testHeader.cc.in from this seemingly arbitrary location to the target that actually uses testHeader.cc.in?

Reply:
Unfortunately not. See comment there.